src/image/Mask.cc

```diff
@@ -392,7 +391,7 @@ namespace {
  */
 template<typename MaskPixelT>
 void Mask<MaskPixelT>::_initializePlanes(MaskPlaneDict const& planeDefs) {
-    pexLog::Trace("afw.Mask", 5, boost::format("Number of mask planes: %d") % getNumPlanesMax());
+    LOGLS_DEBUG("afw.Mask", "Number of mask planes: " << getNumPlanesMax());
```

Comment:
Oh, and while this might be outside the scope of the original ticket, it seems like the log name ought to be `afw.image.Mask`, and this might be a good time to clean that up (both here and in the afw tests that enable this logger).